Toolkit/NewAppDeployToolkit/Functions/Main/Copy-File.ps1

```diff
@@ -27,102 +26,100 @@ Function Copy-File {
 	http://psappdeploytoolkit.com
 #>
 	[CmdletBinding()]
-	Param (
+	param (
```

Comment:
I tend to like to seperate each param with a newline to seperate them so each individual param is easier to read and understand. But that's just me